styles/index.module.scss

```diff
@@ -0,0 +1,522 @@
+$gray-200: rgba(237, 242, 247, 1);
+$gray-300: #E2E8F0;
+$gray-400: #CBD5E0;
+$gray-500: #A0AEC0;
+$gray-600: #718096;
+$gray-700: #4A5568;
+$gray-800: #2D3748;
+
+$yellow-400: #F6AD55;
+
+$teal-400: #71E8DF;
+$teal-500: #04C8BB;
+$teal-800: #154F47;
+
+$red-200: #FED7D7;
+$red-500: #F56565;
+$red-800: #9B2C2C;
+
+$green-800: #22543D;
+$desktop-small: 940px;
+$tablet-vertical: 768px;
+$desktop-large: 1428px;
+$tablet-horizontal: 1024px;
+$mobile-large: 390px;
+
+.main {
+  font-family: "Inter", system-ui, -apple-system, BlinkMacSystemFont,
+  "Segoe UI";
+  margin: 0 auto;
+  max-width: unset;
+}
+
+.h2 {
+  color: white;
+  margin-bottom: 40px;
+  margin-top: 16px;
+}
+
+.waitlistbtn {
+  background-color: transparent !important;
+}
+
+.header {
+  margin: 45px auto 44px auto;
+  max-width: 682px;
+  text-align: center;
+  p {
+    color: $gray-300;
+    max-width: 590px;
+    margin: 0 auto;
+  }
+  .eyebrow-headline {
+    color: $gray-600;
+    /* DESKTOP/HEADINGS/H5 */
+    font-size: 20px;
+    font-family: Barlow;
+    font-weight: 700;
+    line-height: 100%;
+    letter-spacing: 2px;
+    text-transform: uppercase;
+    text-decoration: none;
+    display: flex;
+    gap: 16px;
+    justify-content: center;
+    align-items: flex-end;
+    align-items: center;
+  }
+}
+
+.githubIcon {
+  margin-top: 4px;
+}
+
+.squareIcon {
+  width: 42px;
+  height: 42px;
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: $gray-800;
+}
+
+.infoText {
+  > span {
+    color: $gray-400;
+    margin-bottom: 4px;
+    font-weight: 600;
+  }
+  > div {
+    font-weight: 700;
+    color: white;
+  }
+}
+
+.pageInfo {
+  display: flex;
+  margin-bottom: 32px;
+  max-width: 1052px;
+  width: calc(100% - 64px);
+  margin: 0 auto;
+  flex-wrap: wrap;
+  gap: 44px;
+  justify-content: center;
+  padding: 0 32px 60px;
+  @media (min-width: $tablet-horizontal) {
+    padding: 24px 32px 40px;
+    justify-content: space-between;
+  }
+
+  > div {
+    min-width: 290px;
+    display: flex;
+    gap: 23px;
+    width: calc((100% / 3) - (88px / 3));
+    justify-content: flex-start;
+  }
+}
+
+.badgeYellow {
+  padding: 4px 7px;
+  width: auto;
+  border-radius: 7px;
+  background-color: $gray-800;
+  font-size: 12px;
+  color: $yellow-400;
+  > img {
+    margin-right: 4px;
+  }
```

Comment:
This margin only works when there is text on the side so I think it should be added to the text instead.